contracts/exchange/contracts/src/interfaces/IProtocolFees.sol

```diff
@@ -36,4 +36,16 @@ contract IProtocolFees {
     /// @param updatedProtocolFeeCollector The updated protocolFeeCollector contract address.
     function setProtocolFeeCollectorAddress(address updatedProtocolFeeCollector)
         external;
+
+    /// @dev Returns the protocolFeeMultiplier
+    function protocolFeeMultiplier()
```

Comment:
Nice!

Reply:
Credit to @abandeali1 for that one :smile: